substrate/client/db/src/bench.rs

```diff
@@ -45,16 +42,16 @@ use std::{
 	sync::Arc,
 };
 
-type State<B> = DbState<B>;
+type State<H> = DbState<H>;
 
-struct StorageDb<Block: BlockT> {
+struct StorageDb<Hasher: Hash> {
 	db: Arc<dyn KeyValueDB>,
-	_block: std::marker::PhantomData<Block>,
+	_phantom: std::marker::PhantomData<Hasher::Output>,
```

Comment:
```suggestion
	_phantom: std::marker::PhantomData<Hasher>,
```
I did not check if it compile or can be an issue elsewhere (maybe there is a good reason to address Output).